content/docs/state-and-lifecycle.md

```diff
@@ -99,17 +99,17 @@ class Clock extends React.Component {
 }
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/zKRGpo?editors=0010)
+[**CodePen मा प्रयास गर्नुहोस्**](https://codepen.io/gaearon/pen/zKRGpo?editors=0010)
 
-`Clock` is now defined as a class rather than a function.
+`Clock` अब functionको सट्टा classको रूपमा परिभाषित गरिएको छ।
 
-The `render` method will be called each time an update happens, but as long as we render `<Clock />` into the same DOM node, only a single instance of the `Clock` class will be used. This lets us use additional features such as local state and lifecycle methods.
+प्रत्येक चोटि अपडेट हुने बित्तिकै `render` मेथड चल्नेछ, तर जबसम्म हामी उहि DOM नोडमा `<Clock />` रेन्डर गर्दछौं `Clock` classको एक मात्र instance पप्रयोग गरिने छ। यसले हामीलाई थप सुविधाहरू जस्तै लोकल state र लाइफसाइकल मेथडहरू प्रयोग गर्न दिन्छ।
 
-## Adding Local State to a Class {#adding-local-state-to-a-class}
+## Classमा लोकल State थप्ने विधि {#adding-local-state-to-a-class}
```

Comment:
Should we translate local to `स्थानिय`?

Reply:
I guess we can! I also thought of translating but thought of "local storage" and "local state" being used as a key term so felt a bit doubtful. Either way have made the changes.